File: 1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.py
class Solution:
    def arraySign(self, nums: List[int]) -> int:
        #import math
        list1=[]
        for i in nums:
            if i<0:
                list1.append(-1)
            elif i>0:
                list1.append(1)
            else:
                list1.append(0)
        list2=math.prod(list1)
        return list2

File: 1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.java
class Solution {
    public int arraySign(int[] nums) {
        int answer=0;
        for(int n :nums){
            if (n==0){
                return 0;
            }
            if (n<0){
                answer++;
            }
        }
        return answer%2==0?1:-1;
    }
}